README.md

```diff
@@ -4,4 +4,29 @@ Verification bot, currently for MIT 2027 server.
 
 On the future, support will be added for any MIT Discord server, and potentially other authentication methods
 
-To self-host, add your token to `token.txt` and a random string of your choosing on `pepper.txt`
\ No newline at end of file
+To self-host, add your token to `token.txt` and a random string of your choosing on `pepper.txt`
+
+## Running the Bot
+
+To run the bot, you'll need to install [Node 16.9](https://nodejs.org/) or higher. Create a [new Discord bot](https://discord.com/developers/docs/getting-started) and enable [intents](https://discord.com/developers/docs/topics/gateway#gateway-intents). 
+
+First, clone this package and install the dependencies.
+
+```
+git clone https://github.com/sipb/wide-tim.git
+cd wide-tim
+npm install
+```
+
+Next, add your Discord bot token to `token.txt` and a random string to `pepper.txt`.
+
+```
+echo "INSERT_BOT_TOKEN_HERE" >> token.txt
```

Comment:
One issue I did run into was the program wasn't reading the token from the file properly for some reason. The value of the token printed fine when printing the token variable, but it always threw an error at `client.login`. However, if I directly copy-and-pasted the token into the bot.ts file, everything ran fine. Unsure if I'm just running something incorrectly.

Reply:
Changed to printf (since echo adds a newline which was causing the issue) in [f70940f](https://github.com/sipb/wide-tim/pull/10/commits/f70940ff1615c271b7db5cffdbe45603e7e57848).